cuenca_validations/types/general.py

```diff
@@ -9,10 +15,39 @@ def __init__(self, *args, **kwargs):
         sanitize_dict(self)
 
 
+class CJSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        try:
+            encoded = sanitize_item(o)
+        except AttributeError:
+            encoded = super().default(o)
+        return encoded
+
+
 class StrictPositiveInt(StrictInt, PositiveInt):
     """
     - StrictInt: ensures a float isn't passed in by accident
     - PositiveInt: ensures the value is above 0
     """
 
     ...
+
+
+def digits(
+    min_length: Optional[int] = None, max_length: Optional[int] = None
+) -> Type[str]:
+    namespace = dict(min_length=min_length, max_length=max_length)
+    return type('DigitsValue', (Digits,), namespace)
+
+
+class Digits(ConstrainedStr):
```

Comment:
Where is this ever used directly?

Reply:
I removed it from `__init__.py` but if you don't need to specify a `min_length` or `max_length` could be used as:
```py
from cuenca_validations.types.general import Digits

class Class(BaseModel):
    number: Digits
```

It will ensure that `number` has only digits.

Reply:
I say get rid of it—unless it's being used anywhere.

Also ... where are you using `digits()`? I found it better to define the specific type like `Clabe`, `PaymentCardNumber`, etc.

In the case of reference number, for example, I had to change it to an `conint()` since `'0000'` was not acceptable

Reply:
I wanted to remove this: https://github.com/cuenca-mx/cuenca/blob/16a8a710ea626fb57924df8b1c283aacadb2b826/cuenca/validations/transactions.py#L7-L10 which is also here: https://github.com/cuenca-mx/stpmex-python/blob/ce10c4ef80a488437cb6373a4ee6bed652a6ed7f/stpmex/types.py#L37-L40
So I thought it was a good moment to have it only in one place. Do you think I should remove it and keep only the JSON Encoder in this PR?